### DISCUSSION TITLE
Live view won’t connect

### DISCUSSION THREAD

**Discussion by Git0ffMe** [koush/scrypted#734] [Q&A]
Hello all,
I'm running scrypted on docked, on TrueNas scale. I can see the camera previews on the HomeKit dashboard but I can't get live view to connect. I get the error that the device is un reachable. 

Thanks for any help!

**Comment by ggale13** [comment]
I am experiencing the same problem as of IOS update 16.02 using the Ring plugin, RTSP plugin and the Onvif plugin. Snapshots are updating just fine but on the Home app on my iPhone 12 and my M2 Mac Mini I can not get a live stream - just get the spinning wheel trying to load. Interestingly I have a Homecam app on my Apple TV that takes all 9 homekit camera feeds and Live Streams without problems.

If any one out there has a work around I would be greatful.

**Comment by yoyomax80400** [comment]
I am experienced the same, to resolve this issue, i just enable "Rebroadcast Plugin" and this is fixed. 
Reload homekit plugins, leave home app and all work

**Comment by GagikBagd** [comment]
Hello, I have introduced my Reolink camera into Homekit via onvif, but the live images are not displayed correctly, the live image is getting through between. It is displayed correctly in the snapshot plugin, but not in homekit

**Comment by tommywestside** [comment]
for me it works on iPhone but not in the macOS Home app.

**Comment by prashant-bidgely** [comment]
I had the a similar issue and and the rebroadcast plugging was also checked. To fix this i unchecked/checked the rebroadcast option and then reloaded the homekit pluggin and it started to work after that.

FYI i am running scrypted on home assistant with rtsp cameras.
